src/converter/yaml_converter.py

```diff
@@ -94,30 +96,33 @@ def get_stage_explicit_input_dirnames(self, stage):
         return de
 
     def is_initial(self, stage):
```

Comment:
'initial' and 'terminal' are not part of the benchmarking yaml, so this is not used? (rather, `find_initial_and_terminal_nodes()`) is